apps/dashboard/config/configuration_singleton.rb

```diff
@@ -267,11 +272,15 @@ def config
   end
 
   def read_config
-    f = Configuration.config_file
-    YAML.safe_load(File.open(f.to_s)).symbolize_keys
-  rescue => e
-    Rails.logger.error("Can't read or parse #{Configuration.config_file} because of error #{e}")
-    {}
+    files = [Configuration.config_file] + Dir.glob("#{config_directory}/*.y*ml")
```

Comment:
Would it be better to say `/*.y?ml` or some way to say only a single character can exist there or maybe split this into two Globs, one for `.yml` and one for `.yaml`.  Just seems like making it possible for people to load `.y<any set of characters>ml` extensions which is likely not what we want.

Reply:
Yep! I just used `*.yaml` and `*.yml` explicitly now instead.